### PR TITLE
Blazer Checks: Run Cron Job

### PR DESCRIPTION
**Pull request by sampoder**
Closes #3894, this is how https://github.com/hostolab/covidliste roughly did it.

### CODE REVIEW

**Review by maxwofford — APPROVED** [review]
This looks great! I'm fine with this going in but want to confirm it's good on your side @garyhtou

File: config/schedule.yml [hunk 1]
```diff
@@ -127,3 +127,8 @@ stripe_card_nightly_job:
   cron: "0 0 1 * *" # run first day of each month
   class: "DisbursementJob::Distribute128CollectiveFund"
   queue: "default"
+
+blazer_checks_nightly_job:
+  cron: "0   8 * * *"
+  class: "RunBlazerChecksJob"
+  args: { shedule: "1 day" }
```
Comment (cjdenio) [inline config/schedule.yml]:
```suggestion
  cron: "0 8 * * *"
  class: "RunBlazerChecksJob"
  args: { schedule: "1 day" }
```

**Review by cjdenio — CHANGES_REQUESTED** [review]

File: config/schedule.yml [hunk 2]
```diff
@@ -127,3 +127,8 @@ stripe_card_nightly_job:
   cron: "0 0 1 * *" # run first day of each month
   class: "DisbursementJob::Distribute128CollectiveFund"
   queue: "default"
+
+blazer_checks_nightly_job:
+  cron: "0 8 * * *"
+  class: "RunBlazerChecksJob"
```
Comment (cjdenio) [inline config/schedule.yml]:
```suggestion
  class: "BlazerChecksJob"
```

**Review by cjdenio — APPROVED** [review]

**Comment by sampoder** [comment]
wahoo!

From Sam

**Pull request merged**